css/build.css

```diff
@@ -0,0 +1,144 @@
+fl-helper[name="slide"] {
+  padding-bottom: 0;
+  padding-top: 0;
+}
+
+.skip-container {
+  top: 25px;
+  right: 10px;
+  z-index: 9999;
+  position: fixed;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.skip-container .skip-btn {
+  color: #fff;
+  text-shadow: 0 0px 2px #878787;
+  text-decoration: underline;
+  background: transparent;
+  border: none;
+}
+
+.swiper-button-next {
+  position: fixed;
+  top: 49% !important;
+  z-index: 9999;
+}
+
+.swiper-button-prev {
+  position: fixed;
+  top: 49% !important;
+  z-index: 9999;
+}
+
+.swiper-pagination-progressbar {
+  position: fixed !important;
+  top: 15px !important;
+  left: 0 !important;
+  right: 0 !important;
+  z-index: 9999;
+  width: 100% !important;
+}
+
+/* .fl-with-top-menu .swiper-pagination-progressbar {
+  top: 45px !important;
+} */
+
+.swiper-container-initialized .swiper-wrapper {
+  padding-bottom: 40px;
+  min-height: calc(100vh - 85px) !important;
+}
+
+.swiper-container-horizontal.swiper-container-initialized>.swiper-pagination-progressbar {
```

Comment:
This appears to be repeated from the Slider component. There shouldn't be a need for that.